src/utils/index.ts

```diff
@@ -154,6 +154,7 @@ const formatDollarSignificantAmount = (num: number, minDigits: number, maxDigits
 }
 
 // todo: deprecated, use formatDisplayNumber instead
+// e.g: formatDisplayNumber(number, { style: 'decimal', significantDigits: 2 })
```

Comment:
can we add this to prevent usage of this function in the future?
https://stackoverflow.com/questions/60755711/is-it-possible-to-mark-something-as-deprecated-in-typescript

Reply:
nice, I forgot this